### ISSUE TITLE
Wachter implementation

### ISSUE THREAD

**Issue by ninaspreitzer** [interpretml/DiCE#297]
Hey everyone, 

in the paper is mentioned that DiCE is compared to SimpleCF proposed by Wachter, is there the code of generating those Counterfactuals in this repo? 
I am afraid I cannot find it in the files

Thanks so much for the help!
Cheers

**Comment by amit-sharma** [comment]
The method works only for differentiable models like neural networks. It is not included separately, but if you set the diversity loss coefficient to be zero for the tf or pytorch explainers, then you will obtain a method same as theirs. 

If you notice any difficulties, let us know.
